domain/lib/src/model/sharedspace/shared_space_operation_role.dart

```diff
@@ -61,8 +61,20 @@ class SharedSpaceOperationRole {
     SharedSpaceRoleName.ADMIN,
   ];
 
-  static const deleteNodeInSharedSpace = [
+  static const deleteNodeSharedSpaceRoles = [
     SharedSpaceRoleName.ADMIN,
     SharedSpaceRoleName.WRITER
   ];
+
+  static const duplicateNodeSharedSpaceRoles = [
+    SharedSpaceRoleName.ADMIN,
+    SharedSpaceRoleName.WRITER,
+    SharedSpaceRoleName.CONTRIBUTOR
+  ];
+
+  static const renameNodeSharedSpaceRoles = [
```

Comment:
Why you need these roles?

Reply:
I'm referring to the permissions document and I saw it was missing so I added it